compiler/damlc/daml-doc/src/DA/Daml/Doc/Render/Markdown.hs

```diff
@@ -241,3 +85,55 @@ escapeMd = T.pack . concatMap escapeChar . T.unpack
         | otherwise = [c]
 
     shouldEscape = (`elem` ("[]*_~`<>\\&" :: String))
+
+-- | Render fields as a pipe-table, like this:
+-- >  | Field    | Type     | Description
+-- >  | :------- | :------- | :----------
+-- >  | anA      | a        |
+-- >  | another  | a        | another a
+-- >  | andText  | Text     | and text
+renderMdFields :: RenderEnv -> [(RenderText, RenderText, RenderText)] -> [T.Text]
+renderMdFields _ []  = ["(no fields)"]
+renderMdFields env fields = header <> fieldRows
+  where
+    textFields =
+        [ ( renderMdText env name
+          , renderMdText env ty
+          , renderMdText env doc
+          )
+        | (name, ty, doc) <- fields
+        ]
+
+    fLen = maximum $ 5 :
```

Comment:
Is there a specific reason why this has to be `5` or is `5` just somewhat arbitrarily chosen to avoid very short columns?